### PR TITLE
dayentries and date

### PR DESCRIPTION
**Pull request by yvonneadam**
added array and map function for bit more functionality

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,35 +1,43 @@
...
+      date: '20.07.2022',
+    },
+    {
+      text: 'Ich habe Zeit mit meinen Eltern und Großeltern verbracht.',
+      date: '19.07.2022',
+    },
+  ];
+
   return (
     <div>
```
Comment (stefankorder) [inline src/App.js]: You don't need a div tag here because the StyledPage is your outer tag.

File: src/App.js [hunk 2]
```diff
@@ -1,35 +1,43 @@
...
       <StyledPage>
         <StyledHeading>Wofür bist du heute dankbar?</StyledHeading>
+
         <div>
```
Comment (stefankorder) [inline src/App.js]: You don't need a div tag here, style heading wraps your code so you don't need an extra div tag.

File: src/components/Dayentry.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    'Helvetica Neue', sans-serif;
+  font-size: 15px;
+`;
+
+const Dayentry = props => {
+  return (
+    <>
+      <StyledDatum>{props.datum}</StyledDatum>
+      <StyledBox>{props.text}</StyledBox>
+    </>
```
Comment (stefankorder) [inline src/components/Dayentry.js]: You can destructure your object as follows:
```suggestion
const Dayentry = {datum, text} => {
  return (
    <>
      <StyledDatum>{datum}</StyledDatum>
      <StyledBox>{text}</StyledBox>
    </>
```

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! Please take a look at my comments.

File: src/components/Dayentry.js (hunk 3 above)
Comment (yvonneadam) [reply src/components/Dayentry.js]: sometimes, like here or with extra div tags, it helps me for "my own structure/my style" and to keep tracking for later on for me as a beginner. like how "bad" is it? not a good style, but will I get problems later on maybe? @stefankorder

**Pull request merged**